cwms-data-api/src/main/java/cwms/cda/data/dao/JooqDao.java

```diff
@@ -173,6 +175,14 @@ public void accept(org.jooq.Context<?> ctx) {
         };
     }
 
+    protected static Condition filterExact(Field<String> field, String filter) {
+        if (filter == null) {
+            return DSL.trueCondition();
```

Comment:
I think DSL.noCondition is what you want here

Reply:
Oh didn't know that one! Updated